.maintain/chaostest/src/hypervisor/modules/keyring.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Hypervisor module: function to manage private and public keys of all nodes
+ */
+
+const createPair = require('@polkadot/keyring/pair').default
+const {
+  cryptoWaitReady,
+  naclKeypairFromSeed,
+  schnorrkelKeypairFromSeed
+} = require('@polkadot/util-crypto')
+const { u8aToHex } = require('@polkadot/util')
+const stringToU8a = require('@polkadot/util/string/toU8a').default;
```

Comment:
These two lines can be combined.
```suggestion
const { u8aToHex, stringToU8a } = require('@polkadot/util')
```